contrib/agent/src/integration-test/java/io/opencensus/contrib/agent/instrumentation/ThreadInstrumentationIT.java

```diff
@@ -114,7 +115,8 @@ public void execute(Runnable command) {
             // been wrapped in a different context (by automatic instrumentation of
             // Executor#execute), that context will be attached when executing the Runnable.
             Context context2 = Context.current().withValue(KEY, "wrong context");
-            context2.attach();
+            // Work around findbugs warning
+            Preconditions.checkNotNull(context2.attach(), "context2");
```

Comment:
This is just wrong. You  call the attach method before checking for null.

Reply:
Done. (This is not for checking `context2`, rather it's checking the return value of `context2.attach()`. )